server/views/pages/attentionNeededTab.njk

```diff
@@ -0,0 +1,17 @@
+{% from "pages/attentionNeededTabDescription.njk" import attentionNeededTabDescription %}
```

Comment:
Like that you've extracted these all out. 

Guess these are all `/components` rather than `/pages`?

and you could maybe bring the `attentionNeededTabDescription` in to this file. 

maybe as a follow up?